web/src/components/console/graphs/graph-empty.tsx

```diff
@@ -20,7 +20,7 @@ export const GraphEmpty: React.FC<GraphEmptyProps> = ({ height = 180, loading =
         <div className="skeleton-chart" data-test="skeleton-chart" />
       ) : (
         <div className="text-secondary" data-test="datapoints-msg">
-          {t('public~No datapoints found.')}
+          {t('No datapoints found.')}
```

Comment:
if we use it here, we need to initialize the translation namespace in the `useTranslation` hook

Reply:
I updated this in the latest commit.